prdoc/pr_3060.prdoc

```diff
@@ -0,0 +1,16 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add retry mechanics to `pallet-scheduler`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Fixes #3014
```

Comment:
```suggestion
```
The prdocs should be self contained so that we can eventually compile them into a CHANGELOG